health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/FacilityCatchmentMapping.js

```
@@ -1,22 +1,143 @@
+    <React.Fragment>
+      <Header>{t("MICROPLAN_ASSIGN_CATCHMENT_VILLAGES")}</Header>
+      <div className="inbox-search-wrapper">
+        <InboxSearchComposer
+          configs={config}
+        ></InboxSearchComposer>
+      </div>
+
+      {isRootApprover && data?.TotalCount < 1 &&
+        <ActionBar
+          actionFields={[
+            <Button icon="CheckCircle" label={t(`HCM_MICROPLAN_FINALIZE_FACILITY_TO_VILLAGE_ASSIGNMENT`)} onClick={handleActionBarClick} type="button" variation="primary" />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)